cloudify_awssdk/ec2/resources/vpc.py

```diff
@@ -102,6 +113,14 @@ def create(ctx, iface, resource_config, **_):
     utils.update_resource_id(
         ctx.instance, vpc_id)
 
+    modify_vpc_attribute_args = \
+        _.get('modify_vpc_attribute_args')
+    if modify_vpc_attribute_args:
```

Comment:
@EarthmanT Whats is the point of passing this here as part of create ? I mean does it make sense to create vpc and then update it afterward ?

Reply:
See above comment - will fix.